Shokofin/Providers/MovieProvider.cs

```diff
@@ -0,0 +1,134 @@
+                var filename = Path.Join(
+                        Path.GetDirectoryName(info.Path)?.Split(Path.DirectorySeparatorChar).LastOrDefault(),
+                        Path.GetFileName(info.Path));
+
+                _logger.LogInformation($"Shoko Scanner... Getting movie ID ({filename})");
+
+                var apiResponse = await ShokoAPI.GetFilePathEndsWith(filename);
+                var file = apiResponse?.FirstOrDefault();
+                var fileId = file?.ID.ToString();
+                var seriesIds = file?.SeriesIDs.FirstOrDefault();
+                var seriesId = seriesIds?.SeriesID.ID.ToString();
+                var episodeIds = seriesIds?.EpisodeIDs?.FirstOrDefault();
+                var episodeId = episodeIds?.ID.ToString();
+
+                if (string.IsNullOrEmpty(episodeId) || string.IsNullOrEmpty(seriesId))
+                {
+                    _logger.LogInformation($"Shoko Scanner... File not found! ({filename})");
+                    return result;
+                }
+
+                _logger.LogInformation($"Shoko Scanner... Getting movie metadata ({filename} - {episodeId})");
+
+                var seriesAniDB = await ShokoAPI.GetSeriesAniDb(seriesId);
+                var series = await ShokoAPI.GetSeries(seriesId);
+                var episodeAniDB = await ShokoAPI.GetEpisodeAniDb(episodeId);
+                var episode = await ShokoAPI.GetEpisode(episodeId);
+                bool isMultiEntry = series.Sizes.Total.Episodes > 1;
+                int aniDBId = (isMultiEntry ? episodeIds?.AniDB : seriesIds?.SeriesID.AniDB) ?? 0;
+                int tvdbId = (isMultiEntry ? episodeIds?.TvDB?.FirstOrDefault() : seriesIds?.SeriesID.TvDB?.FirstOrDefault()) ?? 0;
+
+                if (seriesAniDB?.SeriesType != "0")
+                {
+                    _logger.LogInformation($"Shoko Scanner... File found, but not a movie! Skipping.");
+                    return result;
+                }
+
+                var tags = await ShokoAPI.GetSeriesTags(seriesId, Helper.GetFlagFilter());
+                var ( displayTitle, alternateTitle ) = Helper.GetMovieTitles(seriesAniDB.Titles, episodeAniDB.Titles, series.Name, episode.Name, Plugin.Instance.Configuration.TitleMainType, Plugin.Instance.Configuration.TitleAlternateType, info.MetadataLanguage);
+                float comRat = isMultiEntry ? (float)((episodeAniDB.Rating.Value * 10) / episodeAniDB.Rating.MaxValue) : (float)((seriesAniDB.Rating.Value * 10) / seriesAniDB.Rating.MaxValue);
+                ExtraType? extraType = Helper.GetExtraType(episodeAniDB);
+
+                result.Item = new Movie
+                {
+                    IndexNumber = Helper.GetIndexNumber(series, episodeAniDB),
+                    Name = displayTitle,
+                    OriginalTitle = alternateTitle,
+                    PremiereDate = isMultiEntry ? episodeAniDB.AirDate : seriesAniDB.AirDate,
```

Comment:
If it's not a multi-entry, both airdates would be the same. So no need to use a condition here. You can just set it to `episodeAniDB.AirDate`